trillian/integration/hammer.go

```diff
@@ -449,42 +451,51 @@ func (s *hammerState) oneOp(ctx context.Context) error {
 	ep := s.cfg.EPBias.Choose()
 	glog.V(3).Infof("perform %s operation", ep)
 	status := http.StatusOK
+	defer func() {
+		s.stats.done(ep, status)
```

Comment:
Won't this mark the operation as done even on error return?  I guess you could set status to some suitable HTTP error code for the error paths.